pandas/tests/frame/common.py

```diff
@@ -139,3 +142,30 @@ def _check_mixed_int(df, dtype=None):
         assert(df.dtypes['C'] == dtypes['C'])
     if dtypes.get('D'):
         assert(df.dtypes['D'] == dtypes['D'])
+
+
+@contextmanager
+def not_raises(expected_exception: Exception,
+               msg: Optional[str] = None) -> None:
+    """Explicitly checks that a type of exception is not raised inside a
+    with context.
+
```

Comment:
woa what is all this? pls remove

Reply:
Directly addresses the call for explicitness by an earlier comment. I'm well aware a raised exception causes failure and the lack would be success.

Reply:
Removed in 1c35cc70108a0407d964c26b5cfc07d654dd4ebc